### ISSUE TITLE
hello how do I solve this problem

### ISSUE THREAD

**Issue by D4rkkking** [pH7Software/pH7-Social-Dating-CMS#689]
hello how do I solve this problem
![Screenshot_20191216_173714](https://user-images.githubusercontent.com/47460853/70915441-e28f9f80-202a-11ea-88be-3557b32207c2.jpg)

**Comment by Undefined-Variables** [comment]
Have you renamed your .htaccess file from sample.htaccess to .htaccess ???

**Comment by polynamaude** [comment]
@D4rkkking Look at the source of your web pages by right clicking and choosing "View Source". You will see many javascript file and style sheet. Choose one of them and click on it.
You will either see a error or a blank page.
Your cURL is not working properly.
If you want more help than look in the Wiki. You will get more information there.
Starting from there you will also see a page called "How to ask for help". This should give you some hint.

Because with only a screenshot it won't be too easy to investigate ! What's your opinion on this ?

**Comment by D4rkkking** [comment]
deleted and reinstalled, but still the same
![Screenshot_20191221_182900](https://user-images.githubusercontent.com/47460853/71310077-e6ebfc00-2420-11ea-835a-83728d8267af.jpg)

**Comment by weldonpc** [comment]
I have the SAME issue.

When clicking any of the pages at the bottom of my installation for example if you click on "ABOUT US" at www.AmericaDates.us it changes to https:// and adds a "mail." to the front of the domain name. The site switches to what is listed in the original poster's comment and I am stuck.

A few side notes; I have to copy the contents of "pH7Builder-15.3.0" to my root directory before I copy "pH7-Social-Dating-CMS-master". If I do NOT do this, I get nothing more than a 500 error. 

Any help would be welcome. I hope I was descriptive enough and I hope I wasn't too wordy.

W

**Comment by PublicUser-Admin** [comment]
Hey everybody,

I installed in 3 differents servers and I have the same problem : css not found.

I looked at the source of the page and click on the css links, they do not load.

Thanks for the help

PU

**Comment by Undefined-Variables** [comment]
There are a few things I would check. Firstly the .htaccess file has to be renamed from sample.htaccess to .htaccess 

Then check you have the correct permissions on the /static and the /template folders found at the root directory. 

You would be better off uploading the source code without decompressing it first, so as a zip then unzip it from terminal. This way you can be sure it is uploaded correctly. Unzipping it before hand means you are uploading thousands of files and folders when you can do it in one!

**Comment by Undefined-Variables** [comment]
Also you are getting mail.domain.us so check your DNS record to be sure you have it right, The only reason I can see that happening is if you have the mx record incorrect. you should have one A record pointing to an IP address and the mx records should point at the domain not the IP

**Comment by Undefined-Variables** [comment]
JavaScript Console it telling you all you need to know!


![Annotation 2020-01-02 154334](https://user-images.githubusercontent.com/52648055/71675913-d3bd2280-2d76-11ea-9361-9dcef6895c23.jpg)

**Comment by Undefined-Variables** [comment]
![Annotation 2020-01-02 154334](https://user-images.githubusercontent.com/52648055/71675970-f0595a80-2d76-11ea-9f64-e03a1750127d.jpg)

**Comment by NitWitPea** [comment]
undefined-variables, it appears I have this same problem
file not found when clicking on css files
but why?
I was able to change the url to find it though. after /sit_location/ its /site_location/assets/... instead of /site_location/static...  it finds it there, but the way it display in the url bar with assets in the url makes css come up file not found

where is assets folder supposed to be?

**Comment by Undefined-Variables** [comment]
Yeah this is one of them bugs that has wound me up for sure, there is no trigger it just happens with no warning. Think this is linked to server/environment configuration as I have had a server working and one not both with identical source. 

The assets folder is not a real folder, the content comes from template/static folders and are are routed to /assets in the routes file and also within the templates layout

**Comment by Undefined-Variables** [comment]
@polynamaude Can you spare a few minutes please? I have this error happening again, the long url index.php/module/param1 etc and have run out of ideas so got my host involved and they have taken 36hrs to tell me it might be a cache issue (all off) and then finally they told me they have tried everything and dispite their best efforts they are unable to pin point the issue. 

It always starts...

    index.php/module/action/param1

and ends with the intended page and removing the extra url segments will load the page :/

**Comment by Undefined-Variables** [comment]
here are the calls in last hour pulled from log...

/index.php/module/action/param1/legal/asset/css/asset/signup/user/rss/legal/legal/profile/rss/blog/main/rss/blog/rss/blog
/index.php/module/action/param1/static/PFBC/js/jquery/help/faq
/index.php/module/action/param1/static/PFBC/js/jquery/rss/static/css/js/jquery/smoothness/templates/themes/datelove/img/icon/xml/sitemap/user/xml/sitemap/rss/comment-blog
/index.php/module/action/param1/legal/asset/css/asset/signup/user/rss/legal/legal/profile/rss/blog/main/rss/rss/b/rss/rss/rss/rss/rss/b/rss/blog
/index.php/module/action/param1/static/PFBC/js/jquery/rss/static/css/js/jquery/smoothness/templates/themes/datelove/img/icon/xml/sitemap/user/help/rss/legal/rss/comment-picture
/index.php/module/action/param1/static/PFBC/js/jquery/rss/signup/static/js/jquery/templates/themes/datelove/img/icon/asset/css/legal/help/xml/rss/comment-blog
/index.php/module/action/param1/static/PFBC/js/jquery/rss/static/css/js/jquery/smoothness/templates/themes/datelove/img/icon/xml/sitemap/note/main/xml/sitemap/rss/partners
/index.php/module/action/param1/static/PFBC/js/jquery/rss/static/css/js/jquery/smoothness/templates/themes/datelove/img/icon/xml/sitemap/user/xml/sitemap/rss/forum-topic
/index.php/module/action/param1/rss/static/js/jquery/asset/css/game/category/casino
/index.php/module/action/param1/legal/asset/css/asset/signup/user/rss/legal/legal/profile/rss/blog/main/rss/rss/b/rss/rss/rss/blog/main/rss/comment-picture
/index.php/module/action/param1/asset/css/static/js/jquery/signup/static/css/rss/templates/system/modules/blog/themes/rss/note
/index.php/module/action/param1/static/PFBC/js/jquery/rss/static/css/js/jquery/smoothness/templates/themes/datelove/img/icon/xml/sitemap/user/legal/rss/rss/comment-note
/index.php/module/action/param1/legal/asset/css/asset/static/js/geo/rss/rss/rss/blog
/index.php/module/action/param1/asset/css/static/js/jquery/signup/dating/rss/rss/comment-blog
/index.php/module/action/param1/legal/asset/css/asset/signup/user/rss/legal/legal/profile/rss/blog/main/rss/rss/b/xml/rss/rss/rss/rss/note
/index.php/module/action/param1/legal/asset/css/asset/signup/user/rss/legal/legal/profile/rss/blog/main/rss/rss/b/rss/rss/rss/xml/rss/rss/blog
/index.php/module/action/param1/legal/asset/css/asset/signup/user/rss/legal/legal/profile/rss/blog/main/rss/rss/b/rss/rss/rss/rss/rss/rss/rss/comment-blog
/index.php/module/action/param1/static/PFBC/js/jquery/rss/signup/static/js/jquery/templates/themes/datelove/img/icon/asset/css/legal/legal/user/birthday-people/rss/comment-video
/index.php/module/action/param1/static/PFBC/js/jquery/rss/static/css/js/jquery/smoothness/templates/themes/datelove/img/icon/xml/sitemap/user/legal/invite/home/xml/sitemap/xmllink
/index.php/module/action/param1/legal/asset/css/asset/signup/user/xml/rss/help/static/js/jquery/rss/forum-topic
/index.php/module/action/param1/asset/css/static/js/jquery/signup/asset/rss/comment-note
/index.php/module/action/param1/legal/asset/css/asset/signup/user/rss/legal/legal/profile/rss/blog/main/rss/rss/b/rss/rss/blog/category/rss/note
/index.php/module/action/param1/static/PFBC/js/jquery/rss/static/css/js/jquery/smoothness/templates/themes/datelove/img/icon/xml/sitemap/user/xml/rss/help/game/main/search
/index.php/module/action/param1/legal/asset/css/asset/signup/user/rss/legal/legal/profile/xml/rss/rss/forum-topic
/index.php/module/action/param1/legal/asset/css/asset/signup/forum/forum/legal/user/birthday-people/dating/rss/comment-picture

**Comment by Undefined-Variables** [comment]
Found a solution to this problem and I have suggested a change in code. Its lead developers decision to add or not!

**Comment by Undefined-Variables** [comment]
@D4rkkking Can you try adding this to your .htaccess file please

```RewriteRule ^index\.php/module/action/param1/.*/(.*+)$ /$1 [L,R=301]```
